src/algebra/module/linear_map.lean

```diff
@@ -514,6 +514,11 @@ initialize_simps_projections linear_equiv (to_fun → apply, inv_fun → symm_ap
 
 @[simp] lemma inv_fun_eq_symm : e.inv_fun = e.symm := rfl
 
+@[simp] lemma coe_mk_symm [module R M] [module R M₂]
+  {to_fun inv_fun map_add map_smul left_inv right_inv } :
+  ⇑((⟨to_fun, map_add, map_smul, inv_fun, left_inv, right_inv⟩ : M ≃ₗ[R] M₂).symm) = inv_fun :=
+rfl
```

Comment:
I think this lemma should go below `symm_mk` and be renamed to `coe_symm_mk`, since it is `coe (symm (mk ...))`

I'm surprised the linter doesn't complain that `symm_mk` proves it.

Reply:
Done.